api/src/main/java/org/openmrs/module/fhir2/api/translators/MedicationRequestTranslator.java

```diff
@@ -12,7 +12,7 @@
 import org.hl7.fhir.r4.model.MedicationRequest;
 import org.openmrs.DrugOrder;
 
-public interface MedicationRequestTranslator extends ToFhirTranslator<DrugOrder, MedicationRequest>, UpdatableOpenmrsTranslator<DrugOrder, MedicationRequest> {
+public interface MedicationRequestTranslator extends ToFhirTranslator<DrugOrder, MedicationRequest>, OpenmrsFhirUpdatableTranslator<DrugOrder, MedicationRequest> {
```

Comment:
I'm not sure why you changed the interface type here?

Reply:
@ibacher the reason as to why i changed the interface is because` BaseFhirService` uses `OpenmrsFhirTranslator` type when getting the translator while the` MedicationRequestTranslator `classes uses the `UpdatableOpenmrsTranslator` which brings a incompatibility issues during compiling

Reply:
Good point. Yeah, we actually do want all resource to implement that interface, so I suppose this is the right change to make.